typedoc.json

```diff
@@ -1,5 +1,24 @@
 {
   "out": "./docs",
+  "readme": "none",
+  "exclude": [ "**/node_modules/**"],
   "excludeInternal": true,
-  "readme": "none"
+  "entryPointStrategy": "packages",
+  "entryPoints": [
```

Comment:
Please confirm that it doesn't work without this

Reply:
Yeah, without `entrypoints` it produces new warnings, and this one is specific so it doesn't work correctly:
```
warning The --name option was not specified, and no package.json was found
```